### ISSUE TITLE
Questionable value of declaring `pydap.tests` a namespace package

### ISSUE THREAD

**Issue by ghisvail** [pydap/pydap#69]
Is there a particular reason to declare `pydap.tests` a namespace package? This prevents running the tests as installed from the package (via `python -m pytest --pyargs pydap.tests`), which is a desirable feature from a CI perspective for Linux distributions.

**Comment by jameshiebert** [comment]
Personally, I think that we need to overall *all* of the PyDAP-related namespace packages (and generally get rid of the namespace packages since they're not really used or well supported anymore). There seems to be support (e.g. #34 #35 #39) for integrating all of the extraneous packages (pydap.handlers.csv, pydap.responses.netcdf, etc.) into the mainline distribution. But I think we all need to take some thought and care and figure out how to do that, while still allowing people not not install a slew of dependencies (e.g. NetCDF) if not needed.

**Comment by laliberte** [comment]
@jameshiebert what's the strategy here? Are we waiting for all external packages to be merged before removing the namespace package?

**Comment by jameshiebert** [comment]
I didn't necessarily have a strategy, but I assume that that would be a requirement.

**Comment by laliberte** [comment]
At the moment more responses and handlers are seriously out of date so there might be little consequences in just going forward with the transition.

**Comment by jameshiebert** [comment]
Yeah, that's a good point. It might even be *better* to do it now, so that people stop unwittingly installing handlers/responses that won't work with 3.x Pydap.

**Comment by laliberte** [comment]
I agree. I don't know if that's a good idea but we could move all responses/handlers into the main repository even if they are not currently working. We could then add a warning in each one of them saying that these are not currently implemented and add very basic testing for these warnings. We could then slowly make them available if the demand grows. Is this crazy?
